packages/stark-core/src/modules/http/builder/http-abstract-fetch-resource-request-builder.ts

```diff
@@ -1,10 +1,13 @@
+/* tslint:disable completed-docs*/
```

Comment:
The correct format is: `/* tslint:disable:completed-docs */` otherwise all rules are disabled ;)